pkgs/development/tools/misc/roswell/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit, autoconf, automake, curl }:
+
+stdenv.mkDerivation rec {
+  name = "roswell-${version}";
+  version = "17.9.10.82";
+
+  src = fetchgit {
```

Comment:
Use `fetchFromGitHub`?

``` nix
{
  src = fetchFromGitHub {
    owner = "roswell";
    repo = "roswell";
    rev = "v${version}";
  };
}
```

Also `fetchFromGitHub` should be imported at the top instead of `fetchgit`, and may also need a different `sha256` (not sure).